infra/src/main/kotlin/fi/fta/geoviite/api/frameconverter/v1/FrameConverterDataV1.kt

```diff
@@ -52,6 +33,16 @@ data class FrameConverterStringV1(val value: String) {
     }
 }
 
+data class FrameConverterQueryParamsV1(
+    @JsonProperty("koordinaatisto") val coordinateSystem: Srid = Srid(3067),
+    @JsonProperty("geometriatiedot") val featureGeometry: Boolean = false,
+    @JsonProperty("perustiedot") val featureBasic: Boolean = true,
+    @JsonProperty("lisatiedot") val featureDetails: Boolean = true,
+)
+
+data class FrameConverterCoordinateV1(@JsonIgnore val srid: Srid, override val x: Double, override val y: Double) :
+    IPoint
```

Comment:
Olikohan meillä jo aiemmin olemassa jotain dataluokkaa, jolla kiinnitettäisiin koordinaattijärjestelmä koordinaattiin? Minusta mielekäs rakenne, että koordinaatin mukana kulkisi tieto sen edustamasta koordinaattijärjestelmästä.

Teoriassa tämän voisi myös tehdä geneerisemmin FrameConverterCoordinateV1<SRID>, jolloin esim. funktioiden syöte-/paluuarvoihin voisi kiinnittää ainoastaan tietyn SRIDin tarvittaessa. (Tosin näin reviewin yhteydessä en enää olekaan ihan varma toimisiko tuo edes kotlinissa, JVM käsittelee genericsejä vieläkin hieman epäintuitiivisella tavalla meikäläisen aivoille)

Reply:
`GeometryPoint` ottaa myös SRID:n